source/js/components/petition/floating-label-textarea.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
It would be nice to unify this with `floating-label-input.jsx` as the only difference is really the jsx element type, which we could just throw into the constructor. That said, it does make importing a little more muddled.